docs/how-to/backup-your-databases.md

```diff
@@ -27,6 +27,9 @@ hooks:
         - name: posts
     mongodb_databases:
         - name: messages
+    sqlite_databases:
+        - path: /var/lib/sqlite3/mydb.sqlite
+        - name: mydb
```

Comment:
I think the "-" shouldn't be on this line, right? Since the path and the name are part of the same list mapping.